### PR TITLE
feat(database): mssql basic

### LINKED ISSUE #890: MSSQL
Database tests for MSSQL

### PR DESCRIPTION
**Pull request by divyenduz**
Fixes https://github.com/prisma/e2e-tests/issues/890

### CODE REVIEW

File: databases/mssql/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// Like, postgresql://user:password@localhost:5432/database/schema
```
Comment (janpio) [inline databases/mssql/prisma/schema.prisma]: ?

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -464,6 +465,7 @@ jobs:
       DATABASE_DO_PG_BOUNCER_URL: ${{ secrets.DATABASE_DO_PG_BOUNCER_URL }}
       HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
       DATABASE_HEROKU_PGBOUNCER_URL: ${{ secrets.DATABASE_HEROKU_PGBOUNCER_URL }}
+      DATABASE_URL_DB_MSSQL: ${{ secrets.DATABASE_URL_DB_MSSQL }}
```
Comment (janpio) [inline .github/workflows/test.yaml:468]: What database is this using? I can see no service or docker definition.

File: databases/mssql/prisma/schema.prisma (hunk 1 above)
Comment (divyenduz) [reply databases/mssql/prisma/schema.prisma]: From example maybe?

File: .github/workflows/test.yaml (hunk 2 above)
Comment (divyenduz) [reply .github/workflows/test.yaml:468]: AWS RDS MSSQL (in 1password already).

File: databases/mssql/prisma/schema.prisma (hunk 1 above)
Comment (divyenduz) [reply databases/mssql/prisma/schema.prisma]: Removed

File: .github/workflows/test.yaml (hunk 2 above)
Comment (janpio) [reply .github/workflows/test.yaml:468]: Do we usually test external databases for the basic database tests?
If not, we should switch this to Docker as well to not depend on an external service here.
If testing that service is relevant, we can do that as well of course if there is value in it.

Comment (divyenduz) [reply .github/workflows/test.yaml:468]: Almost (if not all) the tests in this repo so far are on an external Postgres RDS, I just did the same for consistency.

Comment (janpio) [reply .github/workflows/test.yaml:468]: Ah, we don't actually have tests for pure mysql and postgres - that is done in the integration tests.
All good then - but maybe rename it to include `aws-rds` so it is clear what is being tested.

Comment (divyenduz) [reply .github/workflows/test.yaml:468]: Renamed to `mssql-aws-rds`, and the test is green.

Comment (divyenduz) [reply .github/workflows/test.yaml:468]: (firebase-functions failed but that is ofc unrelated to this PR)

**Pull request merged**